### PR TITLE
Fix editorconfig issues

### LINKED ISSUE #281: Issue in .editorconfig file
- [ ] Issue Title is in form `Algo/DS Name [Language]`.
- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.

Currently is says
```
[.{py,java}]
```
should be
```
[*.{py,java}]
```

### PR DESCRIPTION
**Pull request by aviaryan**
Fixes #281 

- Ignored Python from editorconfig check as coala-PEP8 already takes care of that.
- Ignored Golang from editorconfig check because Gofmt and Golint are all that we need.

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

File: counting_sort/CountingSort.java [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+        countOccurences(a, c);
+            for (int i = a.length-1; i >= 0; i--) {	//storing number in ascending order in b
+                    b[c[a[i]]-1] = a[i];	//Stores number to it respective position
+                    c[a[i]]--;		//Decrease c
+            }
+            return b;	//Return sorted array
+    }
+    public static void main(String[] args) {
+        int[] A = new int[10000];
+            int k = 0;
```
Comment (Monal5031) [inline counting_sort/CountingSort.java]: Indentation is not consistent here sir @aviaryan

**Comment by aviaryan** [comment]
@Monal5031 Please check again.

File: exponentiation_by_squaring/exponentiation_by_squaring.c [hunk 2]
```diff
@@ -1,10 +1,10 @@
 #include<stdio.h>
```
Comment (aashutoshrathi) [inline exponentiation_by_squaring/exponentiation_by_squaring.c]: give space between `#include` and `<stdio.h>`

File: euclidean_gcd/euclidean_gcd.c [hunk 3]
```diff
@@ -1,10 +1,10 @@
 #include<stdio.h>
```
Comment (aashutoshrathi) [inline euclidean_gcd/euclidean_gcd.c]: give space between `#include` and `<stdio.h>`

Comment (aviaryan) [inline euclidean_gcd/euclidean_gcd.c]: We can do this in a separate patch as this PR was not responsible for it. 
Also, I am think we can create some regex checks for these kinds of issues.

**Pull request merged**